packages/cozy-mjml/src/components/MJFooter.js

```diff
@@ -48,23 +41,25 @@ class MJFooter extends core.BodyComponent {
   }
 
   cozyHelp(locale) {
-    let blog = "Cozy Cloud's blog"
+    let blog = "Twake Workplace's blog"
     let help = 'Help &amp; support'
+    let link = 'https://help.cozy.io/'
     if (locale == 'fr') {
-      blog = 'Blog de Cozy Cloud'
+      blog = 'Blog de Twake Workplace'
       help = 'Aide et support'
+      link = 'https://support.cozy.io/'
     }
     return `
       <mj-section padding="0">
         <mj-group>
           <mj-column width="49.5%" padding="0">
-            <mj-button href="https://blog.cozy.io" mj-class="primary-link" font-size="12px" align="right">${blog}</mj-button>
+            <mj-button href="https://blog.cozy.io/${locale}/" mj-class="primary-link" font-size="12px" align="right">${blog}</mj-button>
```

Comment:
If the locale is not `fr` or `en`, the reader is redirected to `/fr/$(locale)/` and gets a 404. Maybe we should add a check above and fallback to `en`?

Reply:
The same goes for the store images